src/main/kotlin/frc/robot/subsystems/intake/IntakeIOReal.kt

```diff
@@ -35,9 +40,19 @@ class IntakeIOReal : IntakeIO {
         centerMotor.enableVoltageCompensation(12.0)
         centerMotor.burnFlash()
 
-        angleMotor.configurator.apply(IntakeConstants.MOTOR_CONFIG)
+        val config = TalonFXConfiguration().withMotorOutput(
+            MotorOutputConfigs().withNeutralMode(NeutralModeValue.Brake).withInverted(InvertedValue.Clockwise_Positive)
+        ).withFeedback(
+            FeedbackConfigs().withRotorToSensorRatio(1.0).withSensorToMechanismRatio(IntakeConstants.GEAR_RATIO)
```

Comment:
Something you can do in many places: use `import as` so you'll be able to write `Constants.GEAR_RATIO` instead

Reply:
I don't think that's such a good idea. Might create confusion for people reading this code.

Reply:
I think we can try and revert if needed. I'd argue that `config` is less clear